sdk/ai/Azure.AI.Inference/api/Azure.AI.Inference.net8.0.cs

```diff
@@ -1,5 +1,10 @@
 namespace Azure.AI.Inference
 {
+    public static partial class AIInferenceExtensions
+    {
+        public static Azure.AI.Inference.ChatCompletionsClient GetChatCompletionsClient(this System.ClientModel.Primitives.ConnectionProvider provider) { throw null; }
+        public static Azure.AI.Inference.EmbeddingsClient GetEmbeddingsClient(this System.ClientModel.Primitives.ConnectionProvider provider) { throw null; }
```

Comment:
@dargilco, @jhakulin
we need to decide how to name these extension methods. If we name this method GetEmbeddingsClient it will become the main way to do embeddings with AI Projects. Alternatively, we could inject "Inference" somewhere into the name, e.g. GetInferenceEmbeddingsClient, but then some of the names will get long and uglier.

Reply:
I would leave the names as is for now. These APIs are going away with the move to 1DP and Ephemeral Agent, as "inference" in all its forms will be done by the Agent.